include/boost/geometry/strategies/convex_hull/cartesian.hpp

```diff
@@ -0,0 +1,50 @@
+// Boost.Geometry
+
+// Copyright (c) 2020, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_CONVEX_HULL_CARTESIAN_HPP
+#define BOOST_GEOMETRY_STRATEGIES_CONVEX_HULL_CARTESIAN_HPP
+
+
+#include <boost/geometry/strategies/convex_hull/services.hpp>
+#include <boost/geometry/strategies/detail.hpp>
+#include <boost/geometry/strategy/cartesian/side_robust.hpp>
+
+namespace boost { namespace geometry
+{
+
+namespace strategies { namespace convex_hull
+{
+
+template <typename CalculationType = void>
+class cartesian : strategies::detail::cartesian_base
+{
+public:
+    template <typename Geometry>
+    static auto side(Geometry const&)
```

Comment:
Side strategy may be called for points of the same geometry (e.g. in `is_simple`) or for points of 2 geometries (e.g. in `intersection`) and also in addition to that for some temporary point not existing in any of the geometries. In a place where this is done (e.g. in `get_turns`) the information about the original geometries may be unavailable, only arbitrary 3 points. So it is not clear which `Geometry` and how many of them would have to be passed here.

So I propose to remove the parameter at all and have single side strategy only defined by the strategy.

Does it make sense?

Reply:
Yes, I agree, removed.